README.MD

```diff
@@ -1966,7 +1966,27 @@ Properties:
 
 Predefined expandable properties:
 
-* `instance` - [LocalInstance](src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt) object.
+* `instance` - [LocalInstanceManager](src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstanceManager.kt) object.
+
+An example could be providing additional configuration for the `sling.properties` file when running AEM 6.5 on Java 11:
+
+We need to override only two properties in the final `sling.properties` file: 
+* `org.osgi.framework.system.packages.extra`
+* and `org.osgi.framework.bootdelegation`
+
+To achieve that, add file with your props under `gradle` directory in your project:
+`gradle/instance/local/common/crx-quickstart/conf/sling.properties` 
+
+```properties
+# those properties describe what will be exported in the system bundle - minimal dependency set;
+# in Java 11 system bundle by default exports fewer API classes and sometimes we need to extend
```

Comment:
generally speaking...I am not sure if more opening java / system bundle is recommended approach according to OSGi best practices. more probably, libraries should less require/depend on particular Java packages if it is posssible